src/storage/data_object_storage_registry.rs

```diff
@@ -0,0 +1,150 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use rstd::prelude::*;
+use parity_codec::Codec;
+use parity_codec_derive::{Encode, Decode};
+use srml_support::{StorageMap, StorageValue, decl_module, decl_storage, decl_event, Parameter, dispatch};
+use runtime_primitives::traits::{SimpleArithmetic, As, Member, MaybeSerializeDebug, MaybeDebug};
+use system::{self, ensure_signed};
+use crate::traits::{IsActiveMember, ContentIdExists, ContentHasStorage};
```

Comment:
In https://github.com/Joystream/substrate-runtime-joystream/pull/6/files#diff-44e8606db87b39b048179b1f7af8d244R10 I generalized the `IsActiveMember` to `Members` and grouped several related methods. So there will be some changes here (depending on which order the PRs are merged)

Reply:
Thanks! Changes are fine with me... I sort of need members and staked roles to be merged before I can merge this anyway, as you may have noticed reading this PR :)

Reply:
Updated accordingly, thanks again!